src/vmm/src/logger/logging.rs

```diff
@@ -200,7 +200,7 @@ pub struct LoggerConfig {
 /// the log level filter. It would be a breaking change to no longer support this. In the next
 /// breaking release this should be removed (replaced with `log::LevelFilter` and only supporting
 /// its default deserialization).
-#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
+#[derive(Clone, Debug, PartialEq, Eq, Serialize)]
 pub enum LevelFilter {
     /// [`log::LevelFilter:Off`]
     #[serde(alias = "OFF")]
```

Comment:
I dont think we need these aliases anymore, right?

Reply:
removed it, thanks.